Tensile/KernelWriterAssembly.py

```diff
@@ -10998,7 +11025,7 @@ def globalWriteElements(self, kernel, vectorWidths, elements,
   # bpl = bytes per load op
   ##############################################################################
   def chooseGlobalRead(self, useBuffer, bpl, destVgpr, \
-                       addr0, addr1, soffset, offset, extraFields, hi16=0, comment="load C"):
+                       addr0, addr1, soffset, offset, extraFields, directToLds, dtlUseDestVgpr, hi16=0, comment="load C"):
```

Comment:
I think we can unify these 2 new parameters.
We just need 1 parameter like dtlNoDestVgpr to recogize new dtl format.

Reply:
Yea, my first pass at this change was going to add the lds field inside the choose function, but that ended up being messier. I've posted an update combining these parameters.